src/pages/profiles/Profile.js

```diff
@@ -0,0 +1,207 @@
+                  />
+                </div>
+                <div className="mb-4 block font-raleway">
+                  <Input
+                    htmlFor="dateOfBirth"
+                    labelName="Date Of Birth"
+                    type="date"
+                    name="dateOfBirth"
+                    styles={styles}
+                    defaultValue={dob}
+                    id="dateOfBirth"
+                    disabled={disabled}
+                  />
+                </div>
+                <div className="mb-5 block font-raleway">
+                  <Input
+                    htmlFor="email"
+                    labelName="Email"
+                    type="email"
+                    name="email"
+                    styles={`${styles} border-none`}
+                    defaultValue={email}
+                    id="email"
+                    disabled
+                  />
+                </div>
+              </div>
+              <div className="pl-10 lg:pl-24 xl:pl-24">
+                <div className="mb-4 block font-raleway">
+                  <Input
+                    htmlFor="company"
+                    labelName="Company"
+                    type="text"
+                    name="company"
+                    styles={`${styles} border-none`}
+                    defaultValue={company}
+                    id="company"
+                    disabled
+                  />
+                </div>
+                <div className="mb-4 block font-raleway">
+                  <Input
+                    htmlFor="mobileNumber"
+                    labelName="Mobile Number "
+                    type="text"
+                    name="company"
+                    styles={`${styles} border-none`}
+                    defaultValue={mobileNumber}
+                    id="company"
+                    disabled
+                  />
+                </div>
+                <div className="mb-4 block font-raleway">
+                  <Input
+                    htmlFor="nationalId"
+                    labelName="National ID"
+                    type="text"
+                    name="nationalid"
+                    styles={styles}
+                    defaultValue={id}
+                    id="nationalId"
+                    disabled={disabled}
+                  />
+                </div>
+                <div className="mb-4 block font-raleway">
+                  <Input
+                    htmlFor="address"
+                    labelName="Address"
+                    type="text"
+                    name="address"
+                    styles={styles}
+                    defaultValue={address}
+                    id="address"
+                    disabled={disabled}
+                  />
+                </div>
+              </div>
+            </section>
+            <Button
+              name="Save"
+              id="save"
+              onClick={() => {
+                dispatch(saveInfo());
```

Comment:
Consider adding a button to cancel edit since the edit button can be clicked anytime without any major edits intended. 

A button to cancel would be great.